### ISSUE TITLE
defpattern, OnInit and duplicates

### ISSUE THREAD

**Issue by Stingor** [rathena/rathena#4846]
* **rAthena Hash**: last

* **Server Mode**: 
pre-re

* **Description of Issue**: 
  * Result: can't get patterns to be loaded with OnInit label on duplicates npcs when using @loadnpc or @reloadnpc
  * Expected Result: pattern to be loaded and activated with OnInit label when using duplicates npcs when using @loadnpc or @reloadnpc
  * How to Reproduce:

this works
```
gonryun,152,129,6	script	defpattern	1_F_02,{
	end;
	
L_test:
	dispbottom "a test";
	end;

OnInit:
	defpattern 1, "([^:]+):.*\\stest.*", "L_test";
	activatepset 1;
	end;
}
```
this don't

```
-	script	defpatterntest	1_F_02,{
	end;
	
L_test:
	dispbottom "a test";
	end;

OnInit:
	defpattern 1, "([^:]+):.*\\stest.*", "L_test";
	activatepset 1;
	end;
}
gonryun,152,129,6	duplicate(defpatterntest)	Atest::defpatterntest1	1_F_02
prontera,147,191,6	duplicate(defpatterntest)	Atest::defpatterntest2	1_F_02
```

**Comment by Atemo** [comment]
I did a quick test and the duplicates displayed the message as well. It seems to work for me

**Comment by Stingor** [comment]
that's unfortunate

**Comment by Stingor** [comment]
did you used @loadnpc or a server boot to put the npc on?

**Comment by sader1992** [comment]
I can confirm that with `@reloadnpc` 
the problem is not the command it self , the problem is the duplicate npc
normally `@reloadscript` or map load load the events for each copy of the npc
but with `@reloadnpc` only the original npc load the event.
test script >>
```
gonryun,152,129,6	script	defpatterntest	1_F_02,{
OnInit:
	debugmes "'" + strnpcinfo(3) + "' OnInit is Loaded";
end;
}
gonryun,152,129,6	duplicate(defpatterntest)	Atest::defpatterntest1	1_F_02
prontera,155,170,6	duplicate(defpatterntest)	Atest::defpatterntest2	1_F_02
```
with `@reloadscript`
![image](https://user-images.githubusercontent.com/5981261/80315777-13ae8680-87fa-11ea-8d49-b5d316b71408.png)

with `@reloadnpc`
![image](https://user-images.githubusercontent.com/5981261/80315789-245efc80-87fa-11ea-87b0-0208cffe71a5.png)



not sure if it's related but also something like the quest shop wont load the items with `@reloadnpc`
but i thing the quest shop not the same problem cuz it's npc that calls another npc , and even if you made it to 1 npc the problem will still happen

**Comment by Stingor** [comment]
most of time i deal with it like this, putting this on top of npc script so one click and its loaded
```
	if (.init == 0) {
		defpattern 1, "([^:]+):.*\\stest.*", "L_test";
		activatepset 1;
		.init = 1;
	}
```

**Comment by Lemongrass3110** [comment]
Should be fixed in #5553 please test it.
